2021-06-04 11:44:19,331 INFO    MainThread:26644 [wandb_setup.py:_flush():69] setting env: {}
2021-06-04 11:44:19,331 INFO    MainThread:26644 [wandb_setup.py:_flush():69] setting login settings: {'anonymous': 'allow'}
2021-06-04 11:44:19,332 INFO    MainThread:26644 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_114419-1jyn0vku\logs\debug.log
2021-06-04 11:44:19,332 INFO    MainThread:26644 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_114419-1jyn0vku\logs\debug-internal.log
2021-06-04 11:44:19,333 INFO    MainThread:26644 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000012BA323FB48>
2021-06-04 11:44:19,334 INFO    MainThread:26644 [wandb_init.py:init():369] calling init triggers
2021-06-04 11:44:19,334 INFO    MainThread:26644 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-06-04 11:44:19,334 INFO    MainThread:26644 [wandb_init.py:init():418] starting backend
2021-06-04 11:44:19,334 INFO    MainThread:26644 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-06-04 11:44:19,348 INFO    MainThread:26644 [backend.py:ensure_launched():132] starting backend process...
2021-06-04 11:44:19,409 INFO    MainThread:26644 [backend.py:ensure_launched():137] started backend process with pid: 28908
2021-06-04 11:44:19,409 INFO    MainThread:26644 [wandb_init.py:init():423] backend started and connected
2021-06-04 11:44:19,411 INFO    MainThread:26644 [wandb_init.py:init():465] updated telemetry
2021-06-04 11:44:19,411 INFO    MainThread:26644 [wandb_init.py:init():484] communicating current version
2021-06-04 11:44:20,353 INFO    MainThread:26644 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-06-04 11:44:20,355 INFO    MainThread:26644 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-06-04 11:44:20,613 INFO    MainThread:26644 [wandb_init.py:init():522] starting run threads in backend
2021-06-04 11:44:24,844 INFO    MainThread:26644 [wandb_run.py:_console_start():1565] atexit reg
2021-06-04 11:44:24,845 INFO    MainThread:26644 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-06-04 11:44:24,846 INFO    MainThread:26644 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-06-04 11:44:24,846 INFO    MainThread:26644 [wandb_run.py:_redirect():1479] Redirects installed.
2021-06-04 11:44:24,846 INFO    MainThread:26644 [wandb_init.py:init():546] run started, returning control to user process
2021-06-04 11:44:57,596 INFO    MainThread:26644 [wandb_init.py:_pause_backend():249] pausing backend
2021-06-04 11:44:57,596 INFO    MainThread:26644 [jupyter.py:save_ipynb():265] not saving jupyter notebook
